newIDE/app/src/stories/index.js

```diff
@@ -2637,6 +2637,7 @@ storiesOf('EventsTree', module)
             eventsSheetHeight={500}
             preferences={initialPreferences}
             tutorials={eventsTreeTutorials}
+            onEndEditingEvent={action('end editing event')}
```

Comment:
Sorry to be a pain, but can you take this opportunity to extract the `EventsTree` stories to their own story component? 
It's good to take the habit of cleaning up this huge file 🙈